config/composer.json

```diff
@@ -2,6 +2,7 @@
   "name": "digirati/other-vendors",
   "license": "MIT",
   "require": {
+    "doctrine/orm": "^2.5",
```

Comment:
This `composer.json` is for development only, to get the correct dependencies available